HelpSource/Classes/QuartzComposerView.schelp

```diff
@@ -66,14 +66,33 @@ Open the currently loaded composition in Quartz Composer. You will need to reloa
 EXAMPLES::
 
 code::
-////////// Simple example
+
+////////// Simple example A (fullscreen)
+(
+var width= 1500, height= 1250; // use a resolution higher then the one of your screen
```

Comment:
whitespace issues here and on subsequent lines

i don't know anything about QuartzComposerView, but this seems like a pretty crude approach to fullscreen doesn't it? is there a better way?

Reply:
still seems hacky, is there a way to make a maximized window?